### ISSUE TITLE
Is support for the Connexoon box also possible?

### ISSUE THREAD

**Issue by 2benl** [totalimmersion/com.somfy.tahoma#18]
Hi, first of all thnx for al the effort you put into building the app and communicating with users to add extra functions. A friend is thinking about buying a Tahoma box for his io rolles shutters. We were wondering if using the Connexoon box is an option too since its much cheaper? Looking at Somfy’s website they say they will open up access to both boxes in the summer of 2018.

**Comment by totalimmersion** [comment]
Hi,

First of all let me say that I've created this app because I own a Tahoma box and wanted to connect it to Homey. Since there wasn't any documentation at all I had to figure it all out by myself. Therefore the only thing that I'm certain of is that it supports Tahoma. Officially it doesn't support Connexoon since there isn't any way for me to test it (since I don't own one). Having said that, I've heard from 1 user that this app also works with Connexoon. But at the moment I can't officially support Connexoon and therefore usage with Connexoon is at your own risk.
I am aware that Somfy is finally opening their API, which should make it significantly easier for me to support new features in Tahoma as well as Connexoon. I have taken a look at the documentation from Somfy, but it seems it's not finalized yet, since it's missing a lot of things. I am hopeful though that within the next couple of months they update their documentation which should enable me to support more features and the Connexoon, without having me to rely on reverse engineering their API.
I hope this clarifies the current state of affairs a bit.

Best,

Lennart
